appyters/lncHUB2/lncHUB2.ipynb

```diff
@@ -38,7 +64,27 @@
    "outputs": [],
    "source": [
     "%%appyter code_exec\n",
-    "query = {{ query }}"
+    "species = {{ query.value[0][\"args\"][\"name\"]|jsonify }}\n",
+    "query = {{ query.value[0] }}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter code_exec\n",
+    "\n",
+    "{% if query.value[0][\"args\"][\"name\"] == \"Mus_musculus\" %}\n",
+    "path = 'mm_'\n",
+    "num_lnc = '11,274'\n",
+    "{% set num_lnc = \"11,274\" %}\n",
+    "{% else %}\n",
+    "path = ''\n",
+    "num_lnc = '18,705'\n",
+    "{% set num_lnc = \"18,705\" %}\n",
+    "{% endif %}"
```

Comment:
This is a bit undesirable, anyway to pull these stats from the matrix you load later?